File: FliGen.Application/Dto/Tour.cs
using FliGen.Application.Dto.Enum;

namespace FliGen.Application.Dto
{
    public class Tour
    {
        public int Id { get; set; }
        public int LeagueId { get; set; }
        public int SeasonId { get; set; }
        public string Date { get; set; }
        public int HomeCount { get; set; }
        public int GuestCount { get; set; }
        public TourStatus TourStatus { get; set; }
    }
}

File: Services/Web/FliGen.Web/ClientApp/src/app/common/tour.ts
import { TourStatus } from "./tourStatus";

export class Tour {
  id: number;
  date: string;
  homeCount: number;
  guestCount: number;
  leagueId: number;
  seasonId: number;
  tourStatus: TourStatus;
  playerRegistered: boolean;
}
